libutils/include/utils/builtin.h

```diff
@@ -29,6 +31,18 @@
 #define BSWAP_WORD(x) __builtin_bswap64(x)
 #endif
 
+#if CONFIG_WORD_SIZE == 32
```

Comment:
let's group the variants together and issue a warning if `CONFIG_WORD_SIZE` is invalid.
```
    #if CONFIG_WORD_SIZE == 32
    #define BSWAP_WORD(x) __builtin_bswap32(x)
    #define CTZ_WORD(x) CTZL(x)
    #define CLZ_WORD(x) CLZL(x)
    #elif CONFIG_WORD_SIZE == 64
    #define BSWAP_WORD(x) __builtin_bswap64(x)
    #define CTZ_WORD(x) CTZLL(x)
    #define CLZ_WORD(x) CLZLL(x)
    #else
    #error "unsupported CONFIG_WORD_SIZE"
    #endif
```